specs/daito/OpenAPI_Design.md

```diff
@@ -0,0 +1,164 @@
+# VMware Ecosystem OpenAPI
+
+**Author(s)**: [zengyingzhe](https://github.com/zengyingzhe)
+
+## Summary
+
+This documentation serves as the design spec for VMware OpenAPI.
+VMware OpenAPI is a framework which provides generic storage related APIs to upper VMware plugins.
+
+## Motivation
+
+VMware is one of the most important and popular virtualization platforms that depends on storage, so it is very necessary to integrate OpenSDS to VMware platform, and this will also benefit for OpenSDS' further development.
+To achieve various purposes, VMware supports multiple types of plugins. All of them rely on storage APIs to interact with backend storage system.
```

Comment:
Does adding below lines will give more clarity?
"Opensds storage provider  integrates opensds with the vmware infrastructure and allow users to view and select  storage pools provisioned through opensds or 3rd party storages."